src/features/unified-share-modal/utils/convertData.js

```diff
@@ -233,3 +233,25 @@ export const convertCollabsResponse = (
 
     return { collaborators };
 };
+
+/**
+ * Convert an enterprise users API response into an array of internal USM contacts.
+ *
+ * @param {UserCollection} contactsAPIData
+ * @param {string|null} currentUserID
+ */
+export const convertContactsResponse = (
+    contactsAPIData: UserCollection,
+    currentUserID: string | null,
+): Array<contactType> => {
+    const { entries = [] } = contactsAPIData;
+
+    // Return all users except for the current user
+    return entries.reduce((filteredContacts, contact) => {
```

Comment:
This could also be written in a more functional style like `entries.filter(...).map(...)`